state-chain/pallets/cf-threshold-signature/src/lib.rs

```diff
@@ -1669,12 +1669,7 @@ where
 	) -> Result<cf_primitives::ThresholdSignatureRequestId, Self::Error> {
 		// Restricting historical signing only to keys we don't have access to via traditional
 		// means:
-		ensure!(
-			CurrentKeyEpoch::<T, I>::get()
-				.map(|current_epoch| epoch_index < current_epoch - 1)
-				.unwrap_or(false),
-			Error::<T, I>::EpochTooRecent
-		);
+		ensure!(epoch_index <= T::EpochInfo::last_expired_epoch(), Error::<T, I>::EpochTooRecent);
```

Comment:
Do we want `epoch_index < T::EpochInfo::last_expired_epoch()` instead? Not sure what `last_expired` means exactly, but if it is simply the previous epoch, our validators should still have access to that key without the need for the governance to intervene.